### DISCUSSION TITLE
Does openmpi need to be recompiled on Hera-Rocky8?

### DISCUSSION THREAD

**Discussion by DavidHuber-NOAA** [JCSDA/spack-stack#1028] [General]
The `openmpi` installation used on Hera for spack-stack is `/scratch1/NCEPDEV/jcsda/jedipara/spack-stack/openmpi-4.1.5` for both Hera-CentOS and Hera-Rocky8.  Should this be recompiled for Rocky8?

Related PR: NOAA-EMC/GSI#715

**Comment by climbfuji** [comment]
@RatkoVasic-NOAA Mentioned to me yesterday that a different openmpi version is needed (provided by the sysadmins). He'll fix this for the existing spack-stack installations and make a PR for develop so that future versions are good, too
